tools/CouchDB_Import_Demographics.php

```diff
@@ -129,7 +129,10 @@ function _generateQuery() {
             $EDCFields = ", c.EDC as EDC";
             $fieldsInQuery .= $EDCFields;
         }
-        $concatQuery = $fieldsInQuery . $tablesToJoin . " WHERE s.Active='Y' AND c.Active='Y' AND ps.study_consent='yes' AND ps.study_consent_withdrawal IS NULL AND c.PSCID <> 'scanner'";
+        if($config->getSetting("useConsent") === "true") {
+            $concatQuery = $fieldsInQuery . $tablesToJoin . " WHERE s.Active='Y' AND c.Active='Y' AND ps.study_consent='yes' AND ps.study_consent_withdrawal IS NULL AND c.PSCID <> 'scanner'";
+        }
+        else $concatQuery = $fieldsInQuery . $tablesToJoin . " WHERE s.Active='Y' AND c.Active='Y' AND c.PSCID <> 'scanner'";
```

Comment:
FYI: It is good practice to have curly brackets in control structures even if they are only one line.

Suggestion:
Make the query at line 135 the default query and concatenate the consent where conditions to it. This way, it will be more explicit which conditions are added for the consent case.

    $concatQuery = $fieldsInQuery . $tablesToJoin . " WHERE s.Active='Y' AND c.Active='Y' AND c.PSCID <> 'scanner'";
    if($config->getSetting("useConsent") === "true") {
        $concatQuery .= "AND ps.study_consent='yes' AND ps.study_consent_withdrawal IS NULL";
    }